samples/bidirectional-chat/csharp/Function.cs

```diff
@@ -80,8 +78,8 @@ public async Task LeaveUserFromGroup([SignalRTrigger]InvocationContext invocatio
             await UserGroups.RemoveFromGroupAsync(userName, groupName);
         }
 
-        [FunctionName(nameof(Disconnect))]
-        public void Disconnect([SignalRTrigger]InvocationContext invocationContext)
+        [FunctionName(nameof(OnDisconnected))]
+        public void OnDisconnected([SignalRTrigger]InvocationContext invocationContext)
```

Comment:
is `OnDisconnectedAsync` a better matching? why `OnConnected` is Task `OnDisconnected` is void?

Reply:
It's decided by customer, and we don't care about the return value. And the name need to be a convention, so I think the one without `Async` is enough.